data/pages-content/humble-data.md

```diff
@@ -1,51 +1,58 @@
 ---
 title: Beginners' Day - Humble Data Workshop
 subtitle:
-  Would you like to learn to code but don’t know where to start? Taking your first steps in programming can seem like an impossible task so we’ve decided to put on a workshop to show 50 beginners how it can be done and share our passion for the world of data science!
+  Would you like to learn to code but don’t know where to start? Would you like to take a peek at Data Science but are not sure how? Taking first steps are very hard. That is why we have decided to host a workshop to teach beginners how to start with Python and Data Science. Join and let's share our passion for the world of data science!
 ---
```

Comment:
If you want to add a custom social card, feel free to add the file to https://github.com/EuroPython/website/tree/main/social-cards. 
You then should add 
`social_card: card-filename.png` (see this as an example: https://raw.githubusercontent.com/EuroPython/website/main/data/pages-content/pyladies-workshop.md)

assets: https://drive.google.com/drive/folders/1cmOLAsNUSO2L8FCYmH5TZS5gon79buPY?usp=sharing

Reply:
I created the social card and added it to the post. I don't have permission to add the file to the GDrive folder. 😭